test/object-store/sync/client_reset.cpp

```diff
@@ -1046,7 +1046,7 @@ TEST_CASE("sync: client reset", "[sync][pbs][client reset][baas]") {
                         realm->cancel_transaction();
                         return value == 6;
                     },
-                    std::chrono::seconds(20));
+                    std::chrono::seconds(20), std::chrono::milliseconds(500));
```

Comment:
Adding a table has become slower, due to the initialization for the string interner. We need to allow a bit more time, before, checking that the after callback has run.